protected/modules/OphTrConsent/migrations/m170907_181843_anaesthetic_type_multiselect.php

```diff
@@ -19,6 +21,19 @@ public function up()
 
         $this->dropForeignKey('et_ophtrconsent_procedure_anaesthetic_type_id_fk', 'et_ophtrconsent_procedure');
 
+        try {//Drop if exists by another name
+            $this->execute('ALTER TABLE `et_ophtrconsent_procedure` DROP FOREIGN KEY `et_ophtrconsent_procedure_anaesthetic_type_id_fk`;');
+            $this->execute('ALTER TABLE `et_ophtrconsent_procedure` DROP INDEX `et_ophtrconsent_procedure_anaesthetic_type_id_fk` ;');
+        }catch (Exception $e){}
+
+        $this->execute('
+          UPDATE  openeyes.et_ophtrconsent_procedure 
+          SET anaesthetic_type_id = 7 
```

Comment:
You may want to explain why "7" and not any of the other numbers

Reply:
Exactly what I was going to say.

Reply:
It's for no anaesthetic